becas/views.py

```diff
@@ -1,3 +1,17 @@
-# from django.shortcuts import render
+from django.shortcuts import render
+from django.contrib.auth.decorators import user_passes_test, login_required
+from perfiles_usuario.utils import is_servicios_escolares
 
-# Create your views here.
+
+@login_required
+@user_passes_test(is_servicios_escolares)
+def reinscription_studies_left(request):
+    """ View to see the list of studies that require to
+    be marked for reinscription and therefore assing a new
+    scolarship for the student.
+    TODO. This is a dummy view, therefore it may
+    be changed in the future.
+
+    """
+
+    return render(request, 'administracion/dashboard_users.html')
```

Comment:
No, but i believe this template is the one that redirects each user to its dashborad, since the view of Studies that need reinscription is the dashboard for a Servicios Escolares user it loads that template